eLearning-courseware/eLearning.jsonld

```diff
@@ -0,0 +1,1271 @@
+        "definition": {
+          "en": "Image map defines an interface wherein users must correctly identify portions of an image by tagging them via image overlay."
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/matching",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Matching"
+        },
+        "definition": {
+          "en": "Learners must match labeled items to the correct bucket."
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/multipleChoice",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Multiple choice"
+        },
+        "definition": {
+          "en": "A single question with multiple answers for the learner to choose from."
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/sortable",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Sortable"
+        },
+        "definition": {
+          "en": "A single question with multiple answers that must be sorted in the correct order."
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/videoBranching",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Video branching"
+        },
+        "definition": {
+          "en": "The learner watches a video and is then asked a question. Depending on how they answer, they are taken to another video or another question."
+        }
+      }
+    ],
+    "templates": [
+      {
+        "id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/templates/coursePassed",
+        "type": "StatementTemplate",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "prefLabel": {
+          "en": "Course passed"
+        },
+        "definition": {
+          "en": "Learner has passed the course."
+        },
+        "rules": [
+          {
+            "location": "$.actor.mbox",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.id",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.name",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.description",
+            "presence": "included"
+          },
+          {
+            "location": "$.result.success",
+            "any": [true]
+          },
+          {
+            "location": "$.result.completion",
+            "any": [true]
+          },
+          {
+            "location": "$.result.score",
```

Comment:
Since there're two ways to indicate score, if there's one that will always/must always be used, you might get even more specific here

Reply:
We expect `result.score` to be a number. I didn't really see an option for saying something like `this rule requires the key at the location to be a number` or within a certain number range.

Reply:
Some times `result.score` will be an object and contain `result.score.max` and `result.score.raw`.